internal/cli/ui.go

```diff
@@ -36,6 +37,35 @@ func (c *UICommand) Run(args []string) int {
 	// Get our API client
 	client := c.project.Client()
 
+	// Get our default context (used context)
+	name, err := c.contextStorage.Default()
+	if err != nil {
+		c.project.UI.Output(clierrors.Humanize(err), terminal.WithErrorStyle())
+		return 1
+	}
+
+	var ctxConfig *clicontext.Config
+	if name != "" {
+		ctxConfig, err = c.contextStorage.Load(name)
+		if err != nil {
+			c.project.UI.Output(clierrors.Humanize(err), terminal.WithErrorStyle())
+			return 1
+		}
+	} else {
+		if allContexts, err := c.contextStorage.List(); len(allContexts) == 0 || err != nil {
+			if err != nil {
+				c.project.UI.Output(clierrors.Humanize(err), terminal.WithErrorStyle())
+				return 1
+			}
+
+			c.ui.Output("\n"+noContextFoundError, terminal.WithWarningStyle())
+			return 1
+		}
+
+		c.ui.Output("\n"+wpNoServerContext, terminal.WithWarningStyle())
+		return 1
+	}
```

Comment:
Nit: I feel like this `if -> else` block isn't controlling the flow but instead doing error checking, since `name == ""` can only result in a scenario where we `return 1`, so let's move the errors into a check and then have the other things just outside of it as normal flow. The benefit of this approach is just the mental context of "we're doing error handling here" vs. "we're making a decision here and need to understand the outcome"

```suggestion
	if name == "" {
		if allContexts, err := c.contextStorage.List(); len(allContexts) == 0 || err != nil {
			if err != nil {
				c.project.UI.Output(clierrors.Humanize(err), terminal.WithErrorStyle())
				return 1
			}

			c.ui.Output("\n"+noContextFoundError, terminal.WithWarningStyle())
			return 1
		}

		c.ui.Output("\n"+wpNoServerContext, terminal.WithWarningStyle())
		return 1
	}
  
	ctxConfig, err = c.contextStorage.Load(name)
	if err != nil {
		c.project.UI.Output(clierrors.Humanize(err), terminal.WithErrorStyle())
		return 1
	}
```

Reply:
That makes sense to me! 👍🏻 I'll push up a fix

Reply:
I didn't end up doing `Commit suggestion` only because I wasn't sure if GitHub would do the right thing formatting wise and didn't wanna get a lint failure. Thank you for showing me the suggestion though! 🙏🏻